march_joint_inertia_controller/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(march_joint_inertia_controller)
+
+add_compile_options(-std=c++14 -Wall -Wextra -Werror)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  pluginlib
+  roscpp
+  std_msgs
+  message_generation
```

Comment:
isn't this one unused?

Reply:
just like _roscpp_?

Reply:
`std_msgs` and `message_generation` are unused. I think `roscpp` is used by `ros::NodeHandle`, so that one should stay.

Reply:
rt!